extension_service/requirements.txt

```diff
@@ -1,5 +1,6 @@
 asyncpg==0.29.0
 fastapi==0.104.1
+firebase_admin==6.2.0
```

Comment:
I think this should be replaced with https://pypi.org/project/async-firebase/? (I think it's the same library you are using)

Reply:
This `async-firebase` is actually a super light-weight wrapper used for android and ios messaging only and is not the same as what we are using. Replaced `firebase-admin` with https://pypi.org/project/google-cloud-firestore/.